### ISSUE TITLE
Remove video in a playlist from video player page

### ISSUE THREAD

**Issue by anthony-robin** [WalsGit/Add2WL-for-Invidious#4]
Hey, here is another idea I got to improve the Invidious playlist feature :)

Currently it is only possible to *add* a video to a playlist from a player video page and we cannot *remove* it:

<img width="276" alt="Capture d’écran 2024-05-07 à 13 56 19" src="https://github.com/WalsGit/Add2WL-for-Invidious/assets/5428510/1b399b46-9f64-44aa-a024-98f5feef6c41">

It would be amazing if this action could acts as a toggle (if video is already in playlist it removes it) instead of adding it a scond time OR if a second form could be added to permit user to remove directly from playlist.

Do you think it is a good case that could be part of your script ?
Thank you :)

**Comment by WalsGit** [comment]
Hi,

it's odd that it's not a default feature in IV. I know that the whole "add to Watch Later" won't be implemented for now by the IV, because they said it needed a change in their database, so that's why I decided to create this script ; but I don't think it should be too hard for them to add the remove option natively. So, first, my suggestion is to open an issue asking for this enhancement with them so that they could look into it.

That being said, that is not a priority form me but I will look into it. I just noticed that my add/remove button doesn't appear in the watch later playlist thumbs in the video page view (when watching a video from your playlist). that would be a way to remove a video from that page using my script, so fixing that would be my priority for now.

Thanks again for your suggestion and feedback.

**Comment by anthony-robin** [comment]
I agree that having this toggle in Invidious core would be better. There is already an [open issue](https://github.com/iv-org/invidious/issues/4365) on the project to implement the feature, but as for now, it is still in pending state.

Good catch on the video page thumb, I didn't notice it before 😄
